test/e2e/lib/async-base-container.js

```diff
@@ -108,14 +109,23 @@ export default class AsyncBaseContainer {
 				}
 			} );
 	}
-	async setABTestControlGroupsInLocalStorage() {
-		const overrideABTests = config.get( 'overrideABTests' );
 
-		const expectedABTestValue = overrideABTests
-			.map( entry => {
-				return '"' + entry[ 0 ] + '":"' + entry[ 1 ] + '"';
-			} )
-			.join( ',' );
+	async setABTestControlGroupsInLocalStorage() {
+		// eslint-disable-next-line prefer-const
+		let expectedABTestValue = [];
+		const abTestList = abTests.default;
+
+		Object.keys( abTestList ).forEach( function( test ) {
```

Comment:
this could use `.map` instead and return directly into `expectedABTestValue`.

```
const expectedABTestValue = Object.keys( abTestList ).map( test => `"${test}_${abTestList[ test ].datestamp}:${abTestList[ test ].defaultVariation}"` )
```

Reply:
Great, thanks for the suggestion!